Source/Core/Core/PowerPC/Jit64Common/Jit64Util.h

```diff
@@ -48,16 +49,16 @@ class FarCodeCache : public Gen::X64CodeBlock
   }
 };
 
-static const int CODE_SIZE = 1024 * 1024 * 32;
+constexpr int CODE_SIZE = 1024 * 1024 * 32;
 
 // a bit of a hack; the MMU results in a vast amount more code ending up in the far cache,
 // mostly exception handling, so give it a whole bunch more space if the MMU is on.
-static const int FARCODE_SIZE = 1024 * 1024 * 8;
-static const int FARCODE_SIZE_MMU = 1024 * 1024 * 48;
+constexpr int FARCODE_SIZE = 1024 * 1024 * 8;
+constexpr int FARCODE_SIZE_MMU = 1024 * 1024 * 48;
```

Comment:
IIRC this constant is also used in the ARM JIT. Feel free to make a copy of it there, as there is no need to allocate the same buffer on x64 and on arm64.